src/pages/index.js

```diff
@@ -167,22 +187,121 @@ export default function IndexPage({ pkgVersion }) {
+
+              if (filter === 'science') {
+                setVisibleIcons(
+                  getChromicons()?.filter((icon) =>
+                    icon?.categories?.includes('science')
+                  )
+                );
+                return;
+              }
+
+              if (filter === 'health') {
+                setVisibleIcons(
+                  getChromicons()?.filter((icon) =>
+                    icon?.categories?.includes('health')
+                  )
+                );
+                return;
+              }
+            }}
+          />
+          <SearchField
+            className="mb-4 w-full sm:px-6 md:px-0 md:mb-0 md:w-auto"
+            inputClassName="w-full md:w-auto"
+            value={searchText}
+            onChange={(e) => {
+              const { tab } = router.query;
+              const search = e.target.value;
+
+              setSearchText(e.target.value);
+
+              const filteredIcons = tab
+                ? getChromicons()?.filter((icon) =>
+                    icon?.categories?.includes(tab)
+                  )
+                : getChromicons();
+
+              if (!search) {
+                setVisibleIcons(filteredIcons);
+                return;
+              }
+
+              setVisibleIcons(
+                filteredIcons?.filter(
+                  (icon) =>
+                    icon.name.toLowerCase().includes(search.toLowerCase()) ||
+                    icon.description
+                      .toLowerCase()
+                      .includes(search.toLowerCase())
+                )
+              );
+            }}
+          />
         </div>
+        {visibleIcons?.length > 0 ? (
+          <div className="grid grid-cols-2 gap-2 px-4 my-4 max-w-6xl mx-auto sm:px-6 lg:px-16 sm:grid-cols-3 md:grid-cols-4 lg:grid-cols-6">
+            {visibleIcons?.map((icon) => {
+              const Icon = icon.reactComponent;
+              return (
+                <Tile
+                  name={icon.name}
+                  key={icon.name}
+                  isOpen={iconInView?.name === icon?.name}
+                  onClick={() => setIconInView(icon)}
+                >
+                  <Icon className="h-6 w-6" />
+                </Tile>
+              );
+            })}
+          </div>
+        ) : (
+          <Transition
+            appear={true}
+            show={true}
+            enter="transition-opacity duration-300 ease-in"
+            enterFrom="opacity-0"
+            enterTo="opacity-100"
+            leave="transition-opacity duration-250 ease-in-out"
+            leaveFrom="opacity-100"
+            leaveTo="opacity-0"
+          >
+            <div className="py-10 text-center font-bold space-y-2">
+              <p>It looks like we don't have an icon for that yet!</p>
+              <a
+                href={`https://github.com/lifeomic/chromicons/issues/new?title=${encodeURIComponent(
+                  `"${searchText}" icon request`
+                )}`}
```

Comment:
This is pretty neat, being able to use query params to the issues URL to fill out sections of the GitHub issue.  There's more we could do here, but this should be enough for us for now.